### PR TITLE
[WIP] Geo json files refactoring

### PR DESCRIPTION
**Pull request by kcht**
approach #1: provide geojson path and column name in column configuration

### CODE REVIEW

File: superset/connectors/sqla/models.py [hunk 1]
```diff
@@ -941,6 +945,19 @@ def query_datasources_by_name(
     def default_query(qry):
         return qry.filter_by(is_sqllab_view=False)
 
+    @classmethod
```
Comment (tomeksabala) [inline superset/connectors/sqla/models.py:948]: Why is that a class method?

File: superset/connectors/sqla/models.py [hunk 2]
```diff
@@ -941,6 +945,19 @@ def query_datasources_by_name(
...
+    def read_geojson(cls, column_name):
+        db_column = db.session.query(TableColumn).filter(TableColumn.column_name == column_name).first()
+        geojson_file = db_column.geojson_file
+        geojson_filter_name_key = db_column.geojson_filter_name_key
+        output = []
+        if geojson_file and geojson_filter_name_key:
+            with open(config['UPLOAD_FOLDER'] + geojson_file) as f:
+                geojson = json.loads(f.read())
+                for feature in geojson['features']:
```
Comment (tomeksabala) [inline superset/connectors/sqla/models.py]: You could consider:
```
output = [{'name': feature['properties']} for feature in geojson['features']]
```

File: superset/connectors/sqla/models.py [hunk 3]
```diff
@@ -941,6 +945,19 @@ def query_datasources_by_name(
...
         return qry.filter_by(is_sqllab_view=False)
 
+    @classmethod
+    def read_geojson(cls, column_name):
+        db_column = db.session.query(TableColumn).filter(TableColumn.column_name == column_name).first()
+        geojson_file = db_column.geojson_file
+        geojson_filter_name_key = db_column.geojson_filter_name_key
+        output = []
+        if geojson_file and geojson_filter_name_key:
+            with open(config['UPLOAD_FOLDER'] + geojson_file) as f:
```
Comment (tomeksabala) [inline superset/connectors/sqla/models.py]: Better use `os.path.join` for path concat https://docs.python.org/3/library/os.path.html#os.path.join

File: superset/connectors/sqla/models.py [hunk 4]
```diff
@@ -643,7 +643,11 @@ def get_sqla_query(  # sqla
                 if col in self.geofitlerable_column_names():
                     def geo_features_gen(values):
                         for value in values:
-                            val_geo_ = app.config.get("active_geo_filters")[col][value]
+                            geojson_contents = self.read_geojson(col)
+                            for line in geojson_contents:
+                                if line['name'] == value:
+                                    val_geo_ = line['geometry']
```
Comment (tomeksabala) [inline superset/connectors/sqla/models.py]: That could be nicely squeezed with a filter I think.

**Review by tomeksabala — COMMENTED** [review]
It looks really cool. Nice job. Left some comments mainly as a pins for a discussion we could have once you have a minute.

File: superset/connectors/sqla/models.py (hunk 1 above)
Comment (kcht) [reply superset/connectors/sqla/models.py:948]: so that we can call it from ChoroplethMap code without having the instance of SqlaTable.

**Review by tomeksabala — APPROVED** [review]
Looks good :)

**Pull request merged**